packages/cli/src/utils.ts

```diff
@@ -61,6 +62,25 @@ export function getMnemonic (mnemonicFile: string): string | undefined {
   return fs.readFileSync(mnemonicFile, { encoding: 'utf8' }).replace(/\r?\n|\r/g, '')
 }
 
+export function getKeystorePath (keystorePath: string): string {
```

Comment:
very confusing:
if path is a valid `./folder` - you return it as is
if path is `./folder/keystore` - you return `folder`

not sure the usecase for that.

Reply:
KeyManager actually takes `workdir` and not the actual keystore file. If given a dir, it looks for `keystore` filename inside. So I resolve that by either getting the keystore file and passing the dirname to the KeyManager, or by getting the dir itself and passing it as-is. Why does it matter that much?